src/modules/jcms_admin/jcms_admin.install

```diff
@@ -72,3 +72,14 @@ function jcms_admin_update_8105() {
 function jcms_admin_update_8106() {
   drupal_flush_all_caches();
 }
+
+/**
+ * Enable profile_switcher module and switch to minimal.
+ */
+function jcms_admin_update_8107() {
+  \Drupal::service('module_installer')->install(['profile_switcher']);
+  drupal_flush_all_caches();
+  \Drupal::service('profile_switcher.profile_switcher')->switchProfile('minimal');
```

Comment:
What effect does running this command have?

One of the steps of CI is to install the site.

See: https://github.com/elifesciences/journal-cms-formula/blob/04a987db7efc0ee3f321197ba3d8acace2ede5d0/salt/journal-cms/init.sls#L293-L309

We may need to discuss an approach that allows us to adapt CI so it will work with these changes.